packages/moonstone/ToggleButton/ToggleButton.js

```diff
@@ -174,16 +174,24 @@ const ToggleButtonBase = kind({
 				c = toggleOffLabel;
 			}
 			return c;
-		}
+		},
+		toggleIndicator: () => <div className={css.toggleIndicator} />
 	},
 
-	render: ({selected, ...rest}) => {
+	render: ({selected, toggleIndicator, ...rest}) => {
 		delete rest.toggleOffLabel;
 		delete rest.toggleOnLabel;
 		delete rest.small;
 
 		return (
-			<Button css={css} data-webos-voice-intent="SetToggleItem" {...rest} aria-pressed={selected} selected={selected} />
+			<Button
+				{...rest}
```

Comment:
This ordering changes the ability to override `data-webos-voice-intent`.

Reply:
Relocated the data attribute.